src/apps/concepts/pages/ViewConceptPage.tsx

```diff
@@ -54,6 +54,10 @@ const ViewConceptPage: React.FC<Props> = ({
     retrieveConcept(url);
   }, [url, retrieveConcept]);
 
+  const fromDictionary = linkedDictionary || dictionaryToAddTo;
+  const conceptsDictionary = linkedDictionary ? `${linkedDictionary}concepts/` : `${dictionaryToAddTo}concepts/`; 
+  const backUrl = fromDictionary ? conceptsDictionary : `${conceptSource}concepts/`;
```

Comment:
A little bit simpler:

```suggestion
  const fromDictionary = linkedDictionary ?? dictionaryToAddTo;
  const backUrl = `${fromDictionary ? fromDictionary : conceptSource}concepts/`
```